pkg/operator/endpoints/deploy.go

```diff
@@ -37,7 +36,35 @@ func Deploy(w http.ResponseWriter, r *http.Request) {
 	ignoreCache := getOptionalBoolQParam("ignoreCache", false, r)
 	force := getOptionalBoolQParam("force", false, r)
 
-	ctx, err := getContext(r, ignoreCache)
+	configBytes, err := files.ReadReqFile(r, "cortex.yaml")
+	if err != nil {
+		RespondError(w, errors.WithStack(err))
+		return
+	}
+
+	if len(configBytes) == 0 {
+		RespondError(w, ErrorFormFileMustBeProvided("cortex.yaml"))
+		return
+	}
+
+	projectBytes, err := files.ReadReqFile(r, "project.zip")
```

Comment:
since this isn't used until `ctx, err := ocontext.New(userconf, projectBytes, ignoreCache)`, does it make sense to move it after the `userconf, err := userconfig.New("cortex.yaml", configBytes, true)` block?